torch/distributed/optim/utils.py

```diff
@@ -24,6 +24,18 @@
     optim.Adamax: _FunctionalAdamax,
 }
 
+
+# Interface to insert a new functional optimizer to functional_optim_map
+# Example::
+#     >>> import the new functional optimizer
+#     >>> from xyz import new_optimizer
+#     >>> from torch.distributed.optim.utils import register_functional_optim
+#     >>>     register_functional_optim(new_optimizer,new_optimizer)
+
+def register_functional_optim(key,optim):
```

Comment:
not sure why linter didn't complain but can we make it `register_functional_optim(key, optim)`